src/styles/mixin.js

```diff
@@ -0,0 +1,15 @@
+import { css } from 'styled-components';
+
+export const flexBox = (jc = 'center', ai = 'center', fd = 'row') => css`
+  display: flex;
+  justify-content: ${jc};
+  align-items: ${ai};
+  flex-direction: ${fd};
+`;
+
+export const positionCenter = () => css`
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+`;
```

Comment:
고정값으로 사용하는 경우 함수가 아니라 그냥 상수로 보내줘도 될 것 같아요

함수로 사용하려면 이런식으로 바꾸는건 어떨까요?

```suggestion
export const positionCenter = (position = 'absolute') => {
  if(position이 absolute나 fixed가 아니라면) return;
  
  return css`
  position: position; // position은 fixed이거나 absolute일 수 있다.
  top: 50%;
  left: 50%;
  transform: translate(-50%, -50%);
  `
 }
```

Reply:
반영했습니다!